### PR TITLE
chore: Rename Left Sidebar

### PR DESCRIPTION
**Pull request by rainandbare**
Renaming LeftSidebar 👍

### CODE REVIEW

File: client/src/components/framework/layoutSkeleton.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import React from "react";
 /* App dependencies */
 import Controls from "common/components/Controls/Controls";
 import Layout from "./layout";
-import LeftSidebarSkeleton from "../leftSidebar/leftSidebarSkeleton";
+import LeftSidebarSkeleton from "../LeftSidebar/LeftSidebarSkeleton";
```
Comment (rainandbare) [inline client/src/components/framework/layoutSkeleton.tsx:8]: I am going to leave LeftSidebarSkeleton where it is for now cause I don't want to conflict with whatever choices you made about RightSidebarSkeleton.

File: client/src/components/LeftSidebar/components/Categorical/components/Category/components/CategoryValue/components/MiniHistogramTemp/MiniHistogram.tsx
Comment (kaloster) [inline client/src/components/LeftSidebar/components/Categorical/components/Category/components/CategoryValue/components/MiniHistogramTemp/MiniHistogram.tsx:1]: Temp

File: client/src/components/LeftSidebar/components/Categorical/components/Category/components/CategoryValue/components/MiniStackedBarTemp/MiniStackedBar.tsx
Comment (kaloster) [inline client/src/components/LeftSidebar/components/Categorical/components/Category/components/CategoryValue/components/MiniStackedBarTemp/MiniStackedBar.tsx:1]: Temp

**Review by kaloster — APPROVED** [review]
🙌🏽

**Pull request merged**